apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/configuration/AuthFilterConfiguration.java

```diff
@@ -39,4 +40,12 @@ public FilterRegistrationBean<ConsumerAuthenticationFilter> openApiAuthenticatio
     return openApiFilter;
   }
 
+  @Bean
+  public FilterRegistrationBean<UserTypeResolverFilter> authTypeResolverFilter() {
+    FilterRegistrationBean<UserTypeResolverFilter> authTypeResolverFilter = new FilterRegistrationBean<>();
+    authTypeResolverFilter.setFilter(new UserTypeResolverFilter());
+    authTypeResolverFilter.addUrlPatterns("/*");
+    return authTypeResolverFilter;
```

Comment:
How to make sure the `UserTypeResolverFilter` always runs after the `ConsumerAuthenticationFilter`?